src/main/scala/rocket/CSR.scala

```diff
@@ -1381,6 +1414,13 @@ class CSRFile(
     }
     if (usingUser) {
       when (decoded_addr(CSRs.mcounteren)) { reg_mcounteren := wdata }
+      when (decoded_addr(CSRs.menvcfg))    { reg_menvcfg.write(wdata) }
```

Comment:
I spend some time to find the path for `xxxh` CSRs, like `mstatush` and `cycleh`. Turned out there was some special utility

https://github.com/chipsalliance/rocket-chip/blob/bd3276d398c50af21812bc054422609491aba3a3/src/main/scala/rocket/CSR.scala#L1573-L1581


I believe the current implementation does not handle `csrw menvcfgh`

I also do not find the path for writing `mstatush`, do you have any clue?

Reply:
> I also do not find the path for writing mstatush, do you have any clue?

Seems like this is part of privileged spec 1.12 (envcfg is also added in 1.12). Is rocket compatible with privileged spec 1.11 only and we need to spend some time upgrading?

Reply:
That snippet is for counters only. We ought to work towards making it compliant with 1.12.

Reply:
Other parts are fine with me now

> That snippet is for counters only. We ought to work towards making it compliant with 1.12.

I think `menvcfgh` and `henvcfgh` are the only problem left.

Reply:
Writes do menvcfgh and menvcfgh should cause no effects at the moment, since FIOM is the low bit.

Reply:
Understood. I assumed that we must have a `decoded_addr(menvcfgh)` somewhere to make it writable. Turned out it is not the case.